build-system/tasks/extension-helpers.js

```diff
@@ -501,7 +504,7 @@ async function buildExtension(
   if (options.bento) {
     await buildBentoExtensionJs(extDir, name, options);
   }
-  await buildExtensionJs(extDir, name, options);
+  await buildExtensionJs(extDir, name, {...options, bento: false});
```

Comment:
nit: don't need to specify `bento: false`. Lack of the key should default to falsy

Reply:
It might have the key, see condition above. I'm removing it to prevent conflict downstream.

Reply:
nit: would be good to parallelize the two:

```js
await Promise.all([
  options.bento && buildExtensionJs(extDir, name, options),
  buildExtensionJs(extDir, name, {...options, bento: false})
])

```